mixer/adapter/redisquota/redisquota_integration_test.go

```diff
@@ -60,7 +60,7 @@ spec:
     maxAmount: 50
     validDuration: 30s
     bucketDuration: 1s
-    rateLimitAlgorithm: __RATE_LIMIT_ALGORITHM__
```

Comment:
is it ok that the rolling window algorithm is dropped ? 
also other testcases?

Reply:
All functionalities were tested on the unit tests.
This integration test is designed to test the adapter is working with the mixer.
I think it is good.